packages/amplication-client/src/Project/ProjectList.scss

```diff
@@ -29,3 +29,20 @@
     }
   }
 }
+
+.new-project {
+  .amp-button {
+    margin: var(--large-spacing) 0;
+    width: 100%;
+    justify-content: center;
+
+    &:hover:not(:disabled) {
+      background-color: var(--dark-purple);
+    }
+
+    &:hover:disabled {
+      color: var(--white);
+      background: var(--black20);
+    }
+  }
+}
```

Comment:
the styles should be inherited from the design-system instead of settings specific style on the button here